pyproject.toml

```diff
@@ -192,7 +192,8 @@ constraint-dependencies = [
     "xgrammar>=0.1.21", # Address CVE GHSA-5cmr-4px5-23pc
 ]
 override-dependencies = [
-    "apex; sys_platform == 'never'"
+    "apex; sys_platform == 'never'",
+    "kaldiio;  sys_platform == 'never'",
```

Comment:
Based on internal convo could we expand this to remove `Levenshtein, kaldiio & distance`. 

In the future we could bump nemo_asr to latest and just skip `LevenShtein` since the others are taken care of in the newer releases.